bigbluebutton-html5/imports/ui/components/video-provider/component.jsx

```diff
@@ -889,6 +890,13 @@ class VideoProvider extends Component {
     if (isAbleToAttach) {
       this.attach(peer, video);
       peer.attached = true;
+
+      if (isLocal) {
+        const { type, name } = getSessionVirtualBackgroundInfo();
+        if (type !== EFFECT_TYPES.NONE_TYPE) {
+          peer.bbbVideoStream.startVirtualBackground(type, name);
```

Comment:
BBBVideoStream's  `startVirtualBackground` method returns a promise that can throw, so some error handling is warrented here.
Example usage: https://github.com/bigbluebutton/bigbluebutton/blob/299107762419c95c23e49fb4b903f850e953aee8/bigbluebutton-html5/imports/ui/components/video-preview/component.jsx#L336-L344.